<launch>

  <arg name="rviz" default="true" />

  <node pkg="loam_velodyne" type="scanRegistration-64E" name="scanRegistration" output="screen">
    <!--remap from="/velodyne_points" to="/kitti/velo/pointcloud"/-->
  </node>

  <node pkg="loam_velodyne" type="laserOdometry" name="laserOdometry" output="screen" respawn="true"/>


  <node pkg="loam_velodyne" type="laserMapping" name="laserMapping" output="screen"/>
  <node pkg="loam_velodyne" type="transformMaintenance" name="transformMaintenance" output="screen"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find loam_velodyne)/rviz_cfg/loam_velodyne.rviz" />
  </group>

  <!--node name="static_tf" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /world /camera_init  20"/-->

</launch>
